### PR TITLE
set up statistic analyse class - instances

### PR DESCRIPTION
**Pull request by iulusoy**

### CODE REVIEW

**Comment by lkeegan** [comment]
@iulusoy should TypeSystem.xml be installed as part of the package? I added a commit to do this as it seemed necessary to run the notebook, but just saw #7 so not sure this is what is desired: https://github.com/ssciwr/moralization/pull/6/commits/a75e7260a681623c934cedadd6fe261ae6fe3888

File: notebooks/DemoNotebook_statistics.ipynb [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    "    \"test_data-trimmed_version_of-Gerichtsurteile-neg-AW-neu-optimiert-BB\",\n",
+    "    [\n",
+    "        \"Appell\",\n",
+    "        \"Adresassat:in\",\n",
+    "        \"Forderer:in\",\n",
+    "        \"soziale Gruppe\",\n",
+    "        \"Benefizient:in\",\n",
+    "        \"Neutral\",\n",
+    "        \"Institution\",\n",
+    "        \"Expression\",\n",
```
Comment (lkeegan) [inline notebooks/DemoNotebook_statistics.ipynb]: This gives an error for me using the test dataset, but if I remove `"Adresassat:in"` and  `"Expression"` (I also don't see these values in the previous plot) then it works - maybe the test dataset changed?

**Review by lkeegan — APPROVED** [review]
Looks good - nice to already see heatmaps of correlations! I made some (hopefully minor) changes to get it to work for me locally, and added error messages for things I stumbled over in the process.

**Comment by iulusoy** [comment]
> @iulusoy should TypeSystem.xml be installed as part of the package? I added a commit to do this as it seemed necessary to run the notebook, but just saw #7 so not sure this is what is desired: [a75e726](https://github.com/ssciwr/moralization/commit/a75e7260a681623c934cedadd6fe261ae6fe3888)

Strange, I thought it was already in the folder, but yes it should be included.

**Pull request merged**